manager/controllers/advisory_systems_test.go

```diff
@@ -178,3 +178,29 @@ func TestAdvisorySystemsTagsUnknown(t *testing.T) { //nolint:dupl
 func TestAdvisorySystemsWrongOffset(t *testing.T) {
 	doTestWrongOffset(t, "/:advisory_id", "/RH-1?offset=1000", AdvisorySystemsListHandler)
 }
+
+func TestAdvisorySystemsSortByOsName(t *testing.T) { //nolint:dupl
+	utils.SkipWithoutDB(t)
+	core.SetupTestEnvironment()
+
+	w := httptest.NewRecorder()
+	req, _ := http.NewRequest("GET", fmt.Sprintf("/RH-1?sort=%v", "os_name"), nil)
```

Comment:
We don't need to use string formatter here, right? Just use simple string `"/RH-1?sort=os_name"`. The same for the second test.